MIGRATION_GUIDE.md

```
@@ -0,0 +1,468 @@
+   
+   # On main (if you pushed test commit)
+   git checkout main
+   git reset --hard origin/main
+   ```
+
+---
+
+## Part 8: Optional - Archive Azure DevOps Repository
+
+### Step 8.1: Archive or Delete Azure DevOps Repo
+**Option A: Archive (Recommended for safety)**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)